SQL/Archive/17.1/2017-04-24-UseOurIssueTrackerPl0x.sql

```diff
@@ -0,0 +1,22 @@
+INSERT IGNORE INTO
+    Config (`ID`, `ConfigID`, `Value`)
+SELECT
+    (
+        SELECT
+            ID
+        FROM
+            Config
+        WHERE
+            ConfigID = tracker.ID
+    ),
```

Comment:
If there is an issue tracker URL *already* specified, this will be the id of the config. This will cause the code to attempt to insert a duplicate key. The `IGNORE` part of the query will, then, stop the whole query from inserting anything.

If there is *no* issue tracker URL specified, this will be `NULL`. Since, the `ID` column of `Config` is an auto-incrementing PK, it will insert the new row with a new ID.